Makefile

```diff
@@ -1,15 +1,15 @@
 .PHONY:	rpm clean source
 
-KAFKA_VERSION ?= 2.3.0
-SCALA_VERSION ?= 2.12
+KAFKA_VERSION ?= 2.6.3
+SCALA_VERSION ?= 2.13
 VERSION = $(shell echo $(KAFKA_VERSION) | sed "s/-/_/")
-BUILD_NUMBER ?= 3
+BUILD_NUMBER ?= 1
 BUILD_METRICS ?= 1
 SOURCE_NAME = kafka_$(SCALA_VERSION)-$(KAFKA_VERSION)
 SOURCE = $(SOURCE_NAME).tgz
 TOPDIR = /tmp/kafka-rpm
 PWD = $(shell pwd)
-URL = $(shell curl -s https://www.apache.org/dyn/closer.cgi/kafka/$(KAFKA_VERSION)/$(SOURCE)?asjson=1 | python -c 'import sys,json; data=json.load(sys.stdin); print data["preferred"] + data["path_info"]')
+URL = https://archive.apache.org/dist/kafka/$(KAFKA_VERSION)/$(SOURCE)
```

Comment:
Don't need the pick closest mirror logic. The download URL changed to archive because this isn't a latest version.